src/Components/Facility/DoctorCapacity.tsx

```diff
@@ -180,8 +180,8 @@ export const DoctorCapacity = (props: DoctorCapacityProps) => {
         }
       }
       handleUpdate();
+      if (btnType == "Save and Exit") handleClose();
```

Comment:
@Pranshu1902 can we make the button types as an enum, something like constants maybe

Reply:
@khavinshankar done!